tests/e2e/specs/modules/adsense/setup-new-user.test.js

```diff
@@ -345,4 +346,59 @@ describe( 'setting up the AdSense module', () => {
 			await expect( '/' ).toHaveValidAMPForVisitor();
 		} );
 	} );
+
+	describe( 'AMP <amp-auto-ads> tag', () => {
+		beforeAll( async () => {
+			await activatePlugin( 'amp' );
+			await activatePlugin( 'e2e-tests-apply-content-filters' );
+		} );
+
+		beforeEach( () => {
+			datapointHandlers.accounts = ( request ) => {
```

Comment:
[Per @felixarntz 's comment](https://github.com/google/site-kit-wp/issues/2111#issuecomment-707305831) it is a bit inefficient to go through the AdSense setup for each AMP mode just to check the tag output.

How about we do the following:
* add a `setupAdSense` utility, similar to `setupAnalytics` in `tests/e2e/utils/setup-analytics.js` since these tests aren't concerned with the module setup at all
* Move this test to a new `amp-auto-ads-tag.test.js` file in the same directory
* Use the new utility to setup AdSense in the `beforeAll` instead of going through setup for each test